__tests__/app.test.js

```diff
@@ -1,7 +1,12 @@
 const endpointsJson = require("../endpoints.json");
-/* Set up your test imports here */
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
 
-/* Set up your beforeEach & afterAll functions here */
+beforeEach(() => seed(testData));
```

Comment:
We will certainly need to seed our database, but having a think about Test DRIVEN Development for this first endpoint... do we need to seed yet?